src/Type/Accessory/AccessoryNonEmptyStringType.php

```diff
@@ -177,6 +177,11 @@ public function toNumber(): Type
 		return new ErrorType();
 	}
 
+	public function toAbsoluteNumber(): Type
+	{
+		return new ErrorType();
```

Comment:
a non-empty string could also contain a string with numbers, so its not an error

Reply:
I understand that but then why does `toNumber()` returns an `ErrorType`? Shouldn't this be changed as well?